taxi/views.py

```diff
@@ -87,3 +88,20 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
```

Comment:
You forgot to implement point 6 of the task and add two buttons in `car_detail.html`